recipes/perfetto/all/conandata.yml

```
@@ -31,3 +34,6 @@ patches:
   "22.1":
     - patch_file: "patches/v22.x/0001-Fix-perfetto-logging-when-PERFETTO_DISABLE_LOG-activ.patch"
       base_path: "source_subfolder"
+  "25.0":
+    - patch_file: "patches/v25.0/0001-MSVC-Fix-narrowing-conversion-error.patch"
+      base_path: "source_subfolder"
```

```suggestion
      base_path: "source_subfolder"
      # source: https://android-review.googlesource.com/c/platform/external/perfetto/+/2057028
```

We've been planning to add some extra info to patches for too long... even if it is just a comment it can be useful for the future.